src/utils.js

```diff
@@ -94,12 +94,36 @@ const emptyTransport = {
     warn: noop
 };
 
+function generateStackId() {
+    let id = '';
+    const chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+
+    for (let i = 0; i < 4; i++) {
+        id += chars.charAt(Math.floor(Math.random() * chars.length));
+    }
+
+    return id;
+}
+
+function stackTransactionId(context) {
+    let stack;
+
+    if (!context.transactionStack || context.transactionStack.length === 0) {
```

Comment:
I think the second part of the `if` is not usefull here as `[...[], 'abcd']` will produce `['abcd']`
Unless you want to make sure it is an array, if so better use `typeof context.transactionStack === 'array'`

Reply:
ok